dashboard/test/ui/features/projects/personal_project_gallery.feature

```diff
@@ -10,7 +10,57 @@ Scenario: Can Toggle to the Public Project Gallery
   Given I am on "http://studio.code.org/projects"
   And I wait until element "#react-personal-projects" is visible
   And element "#public-gallery" is not visible
-  Then I click selector "#uitest-gallery-switcher div:contains(Public Projects)"
-  Then check that I am on "http://studio.code.org/projects/public"
-  And I wait until element "#public-gallery" is visible
-  And element "#react-personal-projects" is not visible
+  Then I navigate to the public gallery via the gallery switcher
+
+Scenario: Can Publish and Unpublish a Project (Button Version)
```

Comment:
I didn't test the chevron version of the A/B experiment.  Given that the button is the clear winner so far and we'll likely keep it for everyone soon, is it worth investing in testing the other variant here? I've testing publishing from the chevron dropdown manually, and it works fine.

Reply:
Nope! If we're gonna be throwing away the chevron publish button, then manual testing in the interim is the way to go!